### PR TITLE
AGP-90: 1Hive DAO Sponsorship Proposal

### PR DESCRIPTION
**Pull request by burrrata**
A proposal to sponsor the 1Hive DAO's ongoing contributions to the Aragon community.

### CODE REVIEW

File: AGPs/AGP-1Hive-Sponsorship.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+These projects are here today because of the 1Hive community. We have taken great care to design, build, and maintain our DAO so that projects like these are possible. We would like to request a sponsorship from Aragon to support the 1Hive DAO. 
+
+This sponsorship is not a request for funding of any of the individual projects listed above. These have been incubated by the 1Hive community and will request additional funding individually when they are ready. We chose this method because want to give the Aragon community more discretion over which projects to support. Our projects will be presented a la carte for ANT holders to review and support individually vs as an all inclusive prix fixe menu. 
+
+This proposal is a sponsorship to allow us to keep growing the 1Hive DAO, reward community members, and incubate more [early ideas](https://github.com/1Hive/ideas/issues). We feel that this is a happy balance between a core team focused on executing a vision and individual projects that can operate with autonomy. 
+
+## Address of the transfer recipient
+
+1Hive Mainnet DAO Agent Address: `address TBD`
```
Comment (izqui) [inline AGPs/AGP-1Hive-Sponsorship.md]: Please input the DAO address or the one person or entity that will provide the DAO address

**Comment by john-light** [comment]
The time now is after the published deadline of October 10, 2019 at 16:00 UTC for finalizing proposals and moving them to Stage V, and this proposal has not yet been finalized. Per [AGP-1](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-1.md#stage-iv-community-review), this proposal is considered to be withdrawn and this pull request will now be closed:

> If you do not finalize the proposal and request to move the proposal to Stage V before Stage V is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage V begins.

**Comment by john-light** [comment]
The Aragon Association Board of Directors has voted to push back the ANV-4 schedule by one week. This PR is being re-opened and is included in the Community Review Period which will now last until October 17, 2019 at 16:00 UTC. Please finalize the draft and leave a comment saying "This draft is final and ready to move to Stage V" before the end of the Community Review Period if you would like this proposal to be reviewed by the Aragon Association in Stage V.

Ref: https://forum.aragon.org/t/anv-4-delay-announcement/1367

File: AGPs/AGP-1Hive-Sponsorship.md (hunk 1 above)
Comment (burrrata) [reply AGPs/AGP-1Hive-Sponsorship.md]: Thank you for catching that. Updated AGP PR accordingly.

**Comment by burrrata** [comment]
This AGP is final, ready to move to Stage V, and ready to review,.

File: AGPs/AGP-1Hive-Sponsorship.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Created: 2019-09-24
+---
+
+# AGP-TBD: 1Hive Sponsorship Proposal
+
+### Summary
+
+[1Hive](https://1hive.org) is a DAO operating on Aragon. We have multiple initiatives in progress, all of which aim to make Aragon more useful for everyone. Our dream is to support the commons so that open source communities can thrive.
+
+We are a relatively young DAO that only came into existence 4 months ago. In that short time we have initiated several projects, one of which has gone on to receive a nest grant.
```
Comment (john-light) [inline AGPs/AGP-1Hive-Sponsorship.md]:
```suggestion
We are a relatively young DAO that only came into existence four months ago. In that short time we have initiated several projects, one of which has gone on to receive a Nest grant. 
```

File: AGPs/AGP-1Hive-Sponsorship.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Summary
+
+[1Hive](https://1hive.org) is a DAO operating on Aragon. We have multiple initiatives in progress, all of which aim to make Aragon more useful for everyone. Our dream is to support the commons so that open source communities can thrive.
+
+We are a relatively young DAO that only came into existence 4 months ago. In that short time we have initiated several projects, one of which has gone on to receive a nest grant. 
+- [Dandelion](https://1hive.org/blog/2019/08/31/dandelion-overview) - MolochDAO on Aragon ([Nest grantee](https://github.com/aragon/nest/issues/168)).
+- [Apiary](https://apiary.1hive.org) - Like a block explorer, but for Aragon DAOs.
+- [Flora](https://devpost.com/software/flora-sidechain-discount-transactions-q8usv3) - Fast, cheap, and secure sidechain for Aragon DAOs.
+- [Conviction Voting](https://forum.aragon.org/t/conviction-voting-on-aragon-cf-dao-proposal/1195) - Continuous signaling of DAOs sentiment for proposal funding.
```
Comment (john-light) [inline AGPs/AGP-1Hive-Sponsorship.md]:
```suggestion
- [Conviction Voting](https://forum.aragon.org/t/conviction-voting-on-aragon-cf-dao-proposal/1195) - Continuous signaling of a DAO's sentiment for proposal funding.
```

File: AGPs/AGP-1Hive-Sponsorship.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Amount of the transfers
+
+75,000 DAI
+
+25,000 ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+Periodic transfers of the amount above, payable every quarter after each ANV has completed.
```
Comment (john-light) [inline AGPs/AGP-1Hive-Sponsorship.md]:
```suggestion
Recurring transfers of the amount above, payable every quarter after each ANV has completed.
```

**Comment by burrrata** [comment]
@john-light Excellent suggestions. Thank you.

This AGP is final, ready to move to Stage V, and ready to review.

**Pull request merged**